.github/workflows/tests.yml

```diff
@@ -428,40 +428,40 @@ jobs:
   # Local Storage Provider
   # ======================================================================
         
-  save-local-cache:
-    runs-on: self-hosted
-    steps:
-      - uses: actions/checkout@v2
-      - name: Cache npm
-        uses: ./
-        with:
-          type: npm
-          version: ${{ github.run_id }}
-          provider: local
-      - name: Install dependencies
-        run: |
-          npm ci
+  #save-local-cache:
```

Comment:
Removing these local tests since I don't have a self-hosted runner at the moment, causing the tests to fail and the badge to show the failure.  I should make these only run when modifying the local storage provider.